tests/test_pronouncing.py

```diff
@@ -65,6 +65,16 @@ def test_rhymes(self):
             'obliquely', 'steakley', 'szekely', 'uniquely', 'weakley',
             'weakly', 'weekley', 'weekly', 'yeakley']
         self.assertEqual(expected, rhymes)
+        # ensure correct behavior for words that have multiple pronunciations
+        rhymes = pronouncing.rhymes("dove")
+        expected = [
+            "above", "belove", "boeve", "bove", "clove", "cove",
+		    "deneuve", "drove", "glove", "gov", "gove", "grove",
+		    "hove", "labauve", "labov", "labove", "love", "nov",
+		    "o'glove", "of", "rove", "shove", "soave", "stove",
+            "strove", "thereof", "throve", "tov", "trove", "vanhove",
+		    "wove"]
+        self.assertEqual(expected, rhymes)
```

Comment:
Looks good. 

Please could you add a newline between each commented block for readability?

(They could even be in their own test methods so only one thing is tested in each.)